src/test/java/seedu/address/testutil/TypicalSalesRecordEntries.java

```diff
@@ -0,0 +1,54 @@
+package seedu.address.testutil;
```

Comment:
hmm, I have also added this file, but differently... Will it affect your tests greatly, if mine could be used instead?

Reply:
okay can, then I will adjust my code to use your TypicalSalesRecordEntries.java file